### PR TITLE
Fix: option to skip chain detection

### PR DESCRIPTION
**Pull request by tewshi**
Frontend part of #7003 #6885

## Checklist

- [x] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

**Comment by lukicenturi** [comment]
@tewshi don't forget to add the changelog

**Comment by LefterisJP** [comment]
> @tewshi don't forget to add the changelog

There is no need. I have added tht in my PR.

**Comment by lukicenturi** [comment]
Can you wrap the selected items inside chip? something like this?
image

File: frontend/app/src/components/settings/general/EvmChainsToIgnoreSettings.vue [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    .toLocaleLowerCase()
+    .includes(queryText.toLocaleLowerCase());
+
+  return nameIncludes || idIncludes;
+};
+</script>
+
+<template>
+  <div>
+    <div class="text-[1rem] mb-3 mt-4">
```
Comment (lukicenturi) [inline frontend/app/src/components/settings/general/EvmChainsToIgnoreSettings.vue:59]: text-base

**Review by lukicenturi — APPROVED** [review]
LGTM!

**Review by lukicenturi — CHANGES_REQUESTED** [review]

File: frontend/app/src/components/settings/general/EvmChainsToIgnoreSettings.vue [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    return false;
+  }
+
+  const nameIncludes = item.name
+    .toLocaleLowerCase()
+    .includes(queryText.toLocaleLowerCase());
+
+  const idIncludes = item.id
+    .toLocaleLowerCase()
+    .includes(queryText.toLocaleLowerCase());
```
Comment (lukicenturi) [inline frontend/app/src/components/settings/general/EvmChainsToIgnoreSettings.vue:51]: put `queryText.toLocaleLowerCase()` to a const

**Review by lukicenturi — APPROVED** [review]
LGTM!

**Pull request merged**